maintainers/maintainer-list.nix

```diff
@@ -2534,6 +2534,12 @@
     githubId = 2817965;
     name = "f--t";
   };
+  f4814n = {
+    email = "me@f4814n.de";
+    github = "f4814";
+    githubId = 11909469;
+    name = "Fabian Geiselhart";
+  };
```

Comment:
commit should be:
```
maintainers: add f4814n
```

Reply:
This commit has already been merged months ago in another PR. So not much I can do there, I guess...